security/cmd/flexvolume/driver/driver.go

```diff
@@ -0,0 +1,515 @@
+// Copyright 2018 Istio Authors
```

Comment:
I noticed this PR puts flexvolume under cmd/, which is reasonable. I will remove the one under node_agent_k8s once this one is submitted.